changelog/dmd.betterc-unwinding.dd

```diff
@@ -0,0 +1,5 @@
+Stack unwinding is now working in `-betterC` mode.
+
+It can now properly use APIs like `backtrace` to generate stack traces, wich is very useful for debugging purposes.
```

Comment:
When you say 'working', it might give the impression that a failing `assert` now gives a stack trace out of the box. Perhaps clarify that it 'works' on a more technical level, that the generated code contains the unwind info needed for e.g. libunwind.